spec/unit/intercom/contact_spec.rb

```diff
@@ -260,6 +260,20 @@
     end
   end
 
+  describe 'archiving' do
+    let(:contact) { Intercom::Contact.new(id: '1', client: client) }
+
+    it 'should successfully archive a contact' do
+      client.expects(:post).with("/contacts/#{contact.id}/archive")
+      contact.archive
+    end
+
+    it 'should successfully unarchive a contact'
```

Comment:
```suggestion
    it 'should successfully unarchive a contact' do
```
This was causing bundler to fail